docs/getting-started/install-guide.md

```diff
@@ -0,0 +1,126 @@
+
+    For 32-bit computers, download the `dcrinstall-darwin-386-v0.7.0` file. <br />
+    For 64-bit computers, download the `dcrinstall-darwin-amd64-v0.7.0` file.
+
+2. Make dcrinstall-darwin-xxxx-vx.x.x an executable within your terminal:
+
+    Navigate to the directory where the dcrinstall file was downloaded using the `cd` command, run chmod with u+x mode on the dcrinstall file, and run the executable that is created. Below is an example of the commands (change directories or filename as needed):
+    
+    `cd ~/Downloads/` <br />
+    `chmod u+x dcrinstall-darwin-amd64-v0.7.0` <br />
+    `./dcrinstall-darwin-amd64-v0.7.0`
+    
+3. The binaries for `dcrd`, `dcrwallet`, `dcrctl`, and `dcrticketbuyer` can then be found in the `~/decred/` directory.
+
+> Linux:
+
+1. Download the correct file:
+
+    For 32-bit computers, download the `dcrinstall-linux-386-v0.7.0` file. <br />
+    For 64-bit computers, download the `dcrinstall-linux-amd64-v0.7.0` file. <br />
+    For 32-bit ARM computers, download the `dcrinstall-linux-arm-v0.7.0` file. <br />
+    For 64-bit ARM computers, download the `dcrinstall-linux-arm64-v0.7.0` file.
+
+2. Make dcrinstall-darwin-xxxx-vx.x.x an executable within your terminal:
+
+    Navigate to the directory where the dcrinstall file was downloaded using the `cd` command, run chmod with u+x mode on the dcrinstall file, and run the executable that is created. Below is an example of the commands (change directories or filename as needed):
+    
+    `cd ~/Downloads/` <br />
+    `chmod u+x dcrinstall-darwin-amd64-v0.7.0` <br />
+    `./dcrinstall-darwin-amd64-v0.7.0` 
+    
+3. The binaries for `dcrd`, `dcrwallet`, `dcrctl`, and `dcrticketbuyer` can then be found in the `~/decred/` directory.
+
+> Windows:
+
+1. Download the correct file:
+
+    For 32-bit computers, download the `dcrinstall-windows-386-v0.7.0.exe` file. <br /> 
+    For 64-bit computers, download the `dcrinstall-windows-amd64-v0.7.0.exe` file. <br />
+
+2.  Run the dcrinstall executable file.
+
+    You can either double click it or run it from the Command Prompt. 
+    
+3. The binaries for `dcrd`, `dcrwallet`, `dcrctl`, and `dcrticketbuyer` can then be found in the `%HOMEPATH%\decred\` directory (usually %HOMEPATH% is `C:\Users\username`).
+
+## Binary Releases
+
+The newest Binary Releases can be found here: [https://github.com/decred/decred-binaries/releases](https://github.com/decred/decred-binaries/releases). With the exception of the `.msi` and `.dmg` files, they are archives of the latest executable binaries for each release. Although most of this will be unzip and go, instructions are provided for Mac, Linux, and Windows below (assumed proficiency for *BSD users).
+
+> OSX/macOS
+
+1. Download the correct file:
+
+    For 32-bit computers, download the `decred-darwin-386-v0.7.0.tar.gz` file. <br />
+    For 64-bit computers, download the `decred-darwin-amd64-v0.7.0.tar.gz` file.
+
+2. Navigate to download location and open the .tar file:
```

Comment:
It really does that?  Ewwww.  Maybe I don't see it since when I use a mac I use Firefox.  It could be a Safari thing (and again ewww, automatically doing anything with downloaded files is such a bad idea).  That changes seems like a good way to cover all of ours bases I think.